index.js

```diff
@@ -1,7 +1,57 @@
+// Students Ana and Robson
+
+console.log("I'm ready!");
+
 // Iteration 1: Names and Input
+let hacker1 = "Robson";
+let hacker2 = "Ana";
 
+console.log(`The driver's name is ${hacker1}`);
+console.log(`The navigator's name is ${hacker2}`);
 
 // Iteration 2: Conditionals
-
+if (hacker1.length > hacker2.length){
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+}
+else if (hacker1.length < hacker2.length){
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+}
+else if (hacker1.length = hacker2.length){
+    console.log(`Wow, you both have equally long names, ${hacker2.length} characters!`);
+}
 
 // Iteration 3: Loops
+let nameUpperSpace = "";
+
+    for (let i = 0; i < hacker1.length; i++){
+        nameUpperSpace += hacker1[i] + " ";
+    }
+    console.log(nameUpperSpace.toUpperCase());
+
+let nameReverse = ""
+
+    for (let i = hacker2.length - 1; i >= 0; i--) {
+        nameReverse += hacker2[i];
```

Comment:
Here you forgot to add a space between each letter like you did in the previous exercise:
`nameReverse += hacker2[i] + " "`